atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -117,6 +94,6 @@ def run(self):
             return
 
         self.log.debug(
-            "for base image '%s' using local image '%s', id '%s'",
-            df_base, build_base, parent_image_ids[df_base]
+            "for base image '%s' using local image '%s'",
```

Comment:
can we rephrase this? it isn't anymore local image, but pullspec with sha

Reply:
rephrased.